packages/web-components/fast-components/src/design-tokens.ts

```diff
@@ -961,5 +920,56 @@ export const neutralLayer4 = create<Swatch>(
 ).withDefault((element: HTMLElement) =>
     neutralLayer4Recipe.getValueFor(element).evaluate(element)
 );
+
+// Deprecated aliases
+
+/** @public @deprecated Use controlCornerRadius */
+export const cornerRadius = controlCornerRadius;
+/** @public @deprecated Use strokeWidth */
+export const outlineWidth = strokeWidth;
+/** @public @deprecated Use focusStrokeWidth */
+export const focusOutlineWidth = focusStrokeWidth;
+/** @public @deprecated Use neutralStrokeDividerRestDelta */
+export const neutralDividerRestDelta = neutralStrokeDividerRestDelta;
+/** @public @deprecated Use neutralFillStrongRestDelta */
+export const neutralFillToggleRestDelta = neutralFillStrongRestDelta;
+/** @public @deprecated Use neutralFillStrongHoverDelta */
+export const neutralFillToggleHoverDelta = neutralFillStrongHoverDelta;
+/** @public @deprecated Use neutralFillStrongActiveDelta */
+export const neutralFillToggleActiveDelta = neutralFillStrongActiveDelta;
+/** @public @deprecated Use neutralFillStrongFocusDelta */
+export const neutralFillToggleFocusDelta = neutralFillStrongFocusDelta;
+/** @public @deprecated Use neutralFillLayerRestDelta */
+export const neutralFillCardDelta = neutralFillLayerRestDelta;
+/** @public @deprecated Use neutralFillStrongRest */
+export const neutralFillToggleRest = neutralFillStrongRest;
+/** @public @deprecated Use neutralFillStrongHover */
+export const neutralFillToggleHover = neutralFillStrongHover;
+/** @public @deprecated Use neutralFillStrongActive */
+export const neutralFillToggleActive = neutralFillStrongActive;
+/** @public @deprecated Use neutralFillStrongFocus */
+export const neutralFillToggleFocus = neutralFillStrongFocus;
+/** @public @deprecated Use neutralFillLayerRest */
+export const neutralFillCard = neutralFillLayerRest;
+/** @public @deprecated Use focusStrokeOuter */
+export const neutralFocus = focusStrokeOuter;
+/** @public @deprecated Use focusStrokeInner */
+export const neutralFocusInnerAccent = focusStrokeInner;
+/** @public @deprecated Use neutralStrokeRest */
+export const neutralOutlineRest = neutralStrokeRest;
+/** @public @deprecated Use neutralStrokeHover */
+export const neutralOutlineHover = neutralStrokeHover;
+/** @public @deprecated Use neutralStrokeActive */
+export const neutralOutlineActive = neutralStrokeActive;
+/** @public @deprecated Use neutralStrokeFocus */
+export const neutralOutlineFocus = neutralStrokeFocus;
+/** @public @deprecated Use neutralStrokeDividerRest */
+export const neutralDivider = neutralStrokeDividerRest;
+/** @public @deprecated Use neutralLayer1 */
+export const neutralLayerL1 = neutralLayer1;
+/** @public @deprecated Use neutralLayer2 */
+export const neutralLayerL2 = neutralLayer2;
+/** @public @deprecated Use neutralLayer3 */
```

Comment:
Can we talk about removing these? It seems a bit odd to me to introduce *new* exports that are already deprecated. If the goal is to ease migration, can we just provide a migration doc that maps v1 assets to v2 assets?

Reply:
Yeah, that was the goal. I was thinking about that too. I realized they were new. I guess what they would allow is for some find and replace around "Behavior.var" and then at least things would compile. I still like the idea I proposed about some sort of migration wrapper, but I guess a doc is fine.